### PR TITLE
extract missing translations

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

File: ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po [hunk 1]
```diff
@@ -3009,6 +3015,11 @@ msgstr ""
 msgid "Skip to site information"
 msgstr ""
 
+#: ckanext/canada/templates/public/user/confirm_delete.html:8
+#, fuzzy
```
Comment (JVickery-TBS) [inline ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po:3019]: @RabiaSajjad need to update this. Sometimes the compile catalog does this. Just removing what is in the `msgstr` and remove the fuzzy should do

File: ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po [hunk 2]
```diff
@@ -3366,6 +3372,11 @@ msgstr "Passer au contenu principal"
 msgid "Skip to site information"
 msgstr "Passer aux informations sur le site"
 
+#: ckanext/canada/templates/public/user/confirm_delete.html:8
+#, fuzzy
```
Comment (JVickery-TBS) [inline ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po:3376]: @RabiaSajjad I THINK this is the correct french phrase?? maybe just add the missing hyphen and remove the fuzzy?

**Review by JVickery-TBS — CHANGES_REQUESTED** [review]
Just the above comments

**Comment by RabiaSajjad** [comment]
> Just the above comments

changes made

**Review by JVickery-TBS — APPROVED** [review]

File: ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po [hunk 3]
```diff
@@ -3009,6 +3015,9 @@ msgstr ""
 msgid "Skip to site information"
 msgstr ""
 
+msgid "Are you sure you want to delete user - {name}?"
+msgstr "Are you sure you want to delete record - {name}?"
```
Comment (JVickery-TBS) [inline ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po:3019]: @RabiaSajjad this should be `Are you sure you want to delete user- {name}?`, not `record`. Silly babel

**Review by JVickery-TBS — CHANGES_REQUESTED** [review]
Sorry one more above!

File: ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po (hunk 3 above)
Comment (wardi) [reply ckanext/canada/i18n/en/LC_MESSAGES/ckanext-canada.po:3019]: or `""` in that case

**Pull request merged**